### PR TITLE
Adding cells cli to path

### PR DESCRIPTION
**Pull request by arana198**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-pydio-cells/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:

Adding cells cli to path

## Benefits of this PR and context:

Able to call cells cli using `cells configure` rather than `/app/cells configure`

### CODE REVIEW

**Comment by Roxedus** [comment]
This PR will not be merged until it also cater for amd64. We would also like it if it did not include another run directive.

**Comment by arana198** [comment]
> This PR will not be merged until it also cater for amd64. We would also like it if it did not include another run directive.

Updated the pr

**Comment by arana198** [comment]
@Roxedus any chance you can look at this PR?

File: Dockerfile [hunk 1]
```diff
@@ -56,11 +56,11 @@ ENV \
   CELLS_WORKING_DIR="/config" \
   GOPATH="/tmp"
 
+COPY --from=buildstage /app/cells /app/
+COPY root/ /
+
 RUN \
+  export PATH=/app/:$PATH && \
```
Comment (Roxedus) [inline Dockerfile]: PATH can, and should be set a a part of ENV in the dockerfile, much like [this](https://github.com/linuxserver/docker-baseimage-alpine/blob/c4951e7021be5ba59d1fb93a6a6fbb7a92ba78a2/Dockerfile#L69)

File: Dockerfile [hunk 2]
```diff
@@ -56,11 +56,11 @@ ENV \
...
   echo "**** install runtime packages ****" && \
   apk add --no-cache \
     openssl
-
-COPY --from=buildstage /app/cells /app/
-
-COPY root/ /
```
Comment (Roxedus) [inline Dockerfile]: Do not reorganize the structure of the dockerfile (the directory does need to exist when getting added to path).

**Review by Roxedus — CHANGES_REQUESTED** [review]

**Review by Roxedus — APPROVED** [review]

**Pull request merged**